mappings/net/minecraft/predicate/item/EnchantmentPredicate.mapping

```diff
@@ -1,13 +1,15 @@
 CLASS net/minecraft/class_2035 net/minecraft/predicate/item/EnchantmentPredicate
 	FIELD field_20687 ARRAY_OF_ANY [Lnet/minecraft/class_2035;
-	FIELD field_9569 enchantment Lnet/minecraft/class_1887;
+	FIELD field_9569 type Lnet/minecraft/class_1887;
```

Comment:
https://minecraft.gamepedia.com/Advancements/Conditions/item
This json field is called `enchantment`